pom.xml

```diff
@@ -356,9 +355,49 @@
                                 <exclude>checkstyle:/src/test/java/org/takes/facets/fork/TkMethodsTest.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/StateAwareInputStream.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/VerboseListTest.java</exclude>
+
+                                <!--
+                                @todo #517:30min Continue fixing FindBugs violations started in #517.
+                                 #517 turned on the find bugs, temporarily excluded the find bugs.
```

Comment:
@sebing please be consistent, in one comment you have three spellings: `FindBugs` (correct one), `find bugs` and `findbugs`.